code/modules/projectiles/guns/energy/cassad.dm

```diff
@@ -1,4 +1,4 @@
-/obj/item/weapon/gun/energy/cassad
+/obj/item/weapon/gun/energy/cassad //TODO: Replace all in-game instances with "/obj/item/weapon/gun/energy/pulse/cassad" once mapmerger is available, and then delete.
```

Comment:
See `maps/map_shims.dm` on how to deal with path changes while the map is locked.

Reply:
Ah, thank you, I've forgotten you've added this tool.